dashboard/app/controllers/application_controller.rb

```diff
@@ -1,3 +1,4 @@
+require 'cdo/date'
```

Comment:
Phil suggested pulling `.strftime('%Q')` out to a `to_milliseconds` helper.  I think `.usec` only gives the microsecond remainder of the current second, not the total elapsed since epoch.